fdbserver/TagPartitionedLogSystem.actor.h

```diff
@@ -31,6 +37,8 @@
 #include "fdbserver/LogProtocolMessage.h"
 #include "flow/actorcompiler.h" // This must be the last #include.
 
+// TODO: Move implementations to TagPartitionedLogSystem.actor.cpp
```

Comment:
I'd keep implementations in .cpp and create .h file with only declarations.

Reply:
Done. What do we usually do with static ACTOR methods? I keep them in header for now.